charts/quantum-serverless/values.yaml

```
@@ -131,9 +131,9 @@ postgresql:
 # PVC
 # ===================
 
+pvcEnable: true
```

```suggestion
createPvc: true
```

nit: this is more explicit about what this option does

---

Solved in [bf42be0](https://github.com/Qiskit-Extensions/quantum-serverless/pull/1299/commits/bf42be08a05504282e9a0c904d1c0cf698679290)